src/dev/impl/DevToys/ViewModels/Tools/Graphic/ImageConverter/ImageConverterToolViewModel.cs

```diff
@@ -0,0 +1,290 @@
+                            }
+                            else
+                            {
+                                HasInvalidFilesSelected = true;
+                            }
+                        }
+                        else
+                        {
+                            HasInvalidFilesSelected = true;
+                        }
+                    }
+                }).ConfigureAwait(false);
+            }
+            catch (Exception) { IsInfoBarOpen = true; }
+        }
+
+        #endregion
+
+        #region SelectFilesBrowseCommand
+
+        public IAsyncRelayCommand SelectFilesBrowseCommand { get; }
+
+        private async Task ExecuteSelectFilesBrowseCommandAsync()
+        {
+            try
+            {
+                await ThreadHelper.RunOnUIThreadAsync(async () =>
+                {
+                    HasInvalidFilesSelected = false;
+
+                    var filePicker = new FileOpenPicker
+                    {
+                        ViewMode = PickerViewMode.List,
+                        SuggestedStartLocation = PickerLocationId.ComputerFolder
+                    };
+
+                    for (int i = 0; i < SupportedFileExtensions.Length; i++)
+                    {
+                        filePicker.FileTypeFilter.Add(SupportedFileExtensions[i]);
+                    }
+
+                    IReadOnlyList<StorageFile> files = await filePicker.PickMultipleFilesAsync();
+                    for (int i = 0; i < files.Count; i++)
+                    {
+                        QueueNewConversionWorkItem(files[i]);
+                    }
+                });
+            }
+            catch (Exception) { IsInfoBarOpen = true; }
+        }
+
+        #endregion
+
+        #region DeleteAllCommand
+
+        public IRelayCommand DeleteAllCommand { get; }
+
+        private void ExecuteDeleteAllCommand()
+        {
+            ConversionWorkQueue.Clear();
+        }
+
+        #endregion
+
+        #region SaveAllCommand
+
+        public IAsyncRelayCommand SaveAllCommand { get; }
+
+        private async Task ExecuteSaveAllCommandAsync()
+        {
+            try
+            {
+                var works = ConversionWorkQueue.ToList();
+
+                var folderPicker = new FolderPicker
+                {
+                    ViewMode = PickerViewMode.List
+                };
+
+                StorageFolder? selectedFolder = await folderPicker.PickSingleFolderAsync();
+
+                if (selectedFolder is not null)
+                {
+                    ICollection<Task> conversionTasks = new List<Task>();
+
+                    foreach (ImageConversionWorkItem work in works)
+                    {
+                        conversionTasks.Add(SaveConversionWorkItem(selectedFolder, work));
+                    }
+
+                    await Task.WhenAll(conversionTasks);
+                }
+            }
+            catch (Exception) { IsInfoBarOpen = true; }
```

Comment:
Let's log the exception, so users can share the issue when a problem occurs :).
https://github.com/veler/DevToys/blob/b8f9e4bb12eccf71da6dc2bc8479f0dd09d09138/src/dev/impl/DevToys/Core/Logger.cs#L60